test/CMakeLists.txt

```diff
@@ -1,13 +1,12 @@
 #
 # CMake script for building the SDL tests
 #
-
```

Comment:
You can also replace `find_package(PythonInterp)` usage with `find_package(Python3)` I think

Reply:
Good suggestion. I appied it. Thanks!